cmd/cmd.go

```diff
@@ -35,6 +40,27 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	dotGauge        = ".gauge"
```

Comment:
There is already a constant for .gauge folder defined in common. We should be using that variable everywhere.